rpc/client/connect.go

```diff
@@ -33,8 +34,17 @@ func (c *Client) openGRPCConn() error {
 
 	var err error
 
+	opts := make([]grpcstd.DialOption, 0, 1)
```

Comment:
We choose a single option, so slice is redundant here. We can do it just through
```
var credOpt grpcstd.DialOption
```

Reply:
i thought that there could be more in the future, but i was not sure bout such decision

fixed